src/core/components/footer/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+import { Footer } from '../../../common/ui-kit'
+
+function PageFooter() {
+  return (
+    <Footer style={{ textAlign: 'center', padding: 0 }}>
```

Comment:
Maybe it would be better if you added a file for example footer.scss for each component imported from antd in the future your styles may be more

Reply:
Further we will add separate scss file for components styling. It is current solution, to be easier to understand how layout looks now.